src/lib/crypto/kimchi_bindings/wasm/README.md

```diff
@@ -0,0 +1,29 @@
+# Kimchi WASM
+
+This code allows us to compile parts of Kimchi into [Web Assembly (WASM)](https://webassembly.org/).
+
+## Requirements
+
+For this to work, you will need to install the following dependencies:
+
+* [wasm-pack](https://rustwasm.github.io/wasm-pack/installer/)
+* [wasm-bindgen-cli](https://rustwasm.github.io/docs/wasm-bindgen/reference/cli.html)
```

Comment:
I don't have the wasm-bindgen-cli installed and I can still build everything. Maybe add an "(optional)" there?

Reply:
sorry I ended up addressing these here: https://github.com/MinaProtocol/mina/pull/11834